packages/generators/commons/src/GeneratorNotificationService.ts

```diff
@@ -6,33 +6,50 @@ export interface GeneratorNotificationService {
 }
 
 export class GeneratorNotificationServiceImpl implements GeneratorNotificationService {
```

Comment:
Let's actually get rid of the interface/impl pattern here -- one rule we want to start imposing is that an `interface` is only relevant if there are more than 1 impls (in this case there aren't)